### PR TITLE
Remove hash urls to issues

### PR DESCRIPTION
**Pull request by nickoneill**
With associated backend changes

### CODE REVIEW

File: static/js/components/issuesListItem.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ module.exports = (issue, state, prev, send) => {
 
   return html`
     <li onclick=${handleClick}>
-      <a aria-controls="content" class="${classString(state, '')}" href="#issue/${issue.id}">
```
Comment (oderby) [inline static/js/components/issuesListItem.js:39]: As discussed online, should keep the href and remove the 'location:set' action from main.js instead.

**Comment by oderby** [comment]
@nickoneill ping - what's blocking here? How can I help? With the release of the "more issues" page, the back button problem is more apparent on the site, so I'd like to get #221 in, in some form. If this is blocked, I say we merge #221 as is (I've just tested it rebased onto current master and it still works correctly) and then remove the onHashChange code here. Thoughts?

**Comment by nickoneill** [comment]
Not blocked, just timing. Working on this now.

**Comment by nickoneill** [comment]
remove in favor of #256

**Pull request closed without merging**